best-practices/estimating-project-maintenance.md

```diff
@@ -0,0 +1,39 @@
+## Maintenance window goal
+
+When we stop actively working on a project, the project moves to the maintenance phase. Any project that does not have a full-time team on it is considered to be on maintenance. The maintenance window goal is to ensure that the project does not go to a state where we would not want to work on it. The maintenance window also serves as a pragmatic way of bulking tasks to reduce context-switch.
+The maintenance window is not a tool for moving a project out of the legacy status nor to develop new functionality.
+
+During a planned maintenance window, the DI team will:
+- Update security vulnerabilities (by Dependabot).
+- Check the Asana board for any reported bugs and fix them.
+- Check error tracking (Rollbar) and fix them.
```

Comment:
For easy fixable errors, fix them.
But in other cases, the client should pay outside of the SLA for the fix, so we would normally create a task for it and inform the client.

Reply:
SLAs are paid per hour. They pay for 30 hours of scheduled work every X amount of time. 
If a task cannot be completed on the scheduled maintenance window, like a major Rails update, or a tricky bug, we will inform the client.

I'll add this to the description to make sure the process is clear.